Components/Searchbar.jsx

```diff
@@ -0,0 +1,11 @@
+export default function Searchbar(){
+    return(
+        <form id="searchBar">
+        <label htmlFor="searchTerm">
+          Search by Article Id
```

Comment:
The API doesn't have any functionality for a search feature, you could implement this on the front end if you wanted to but it isn't part of any core task. 

If you do chose to implement it, I wouldn't recommend using `Article Id` as the search criteria. The ID's are for you as a developer to make use of, they don't mean much to the average user